src/test/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/AlkylStructureFragmenterTest.java

```diff
@@ -53,19 +51,41 @@
  * @version 1.1.1.0
  */
 public class AlkylStructureFragmenterTest {
-    public final File testStructureFile = new File("src/test/resources/ASF_Test_COCONUT_subset_sample.sdf");
-    public final IteratingSDFReader testStructureSDFReader = new IteratingSDFReader(new FileReader(testStructureFile), new SilentChemObjectBuilder());
+    /**
+     * File containing the structure data for the expected fragments of structures used in this test.
+     */
+    public final File testExpectedFragmentsFile = new File("src/test/resources/ASF_Expected_Fragments.sdf");
```

Comment:
You can import this as a resource directly, whithout having to give the full path.